BPA/Ch.08/mr_ss.stan

```diff
@@ -7,9 +15,19 @@ functions {
   }
 
   int last_capture(int[] y_i) {
+  /**
+   * Return an integer value denoting occasion of last capture.
+   * This function is derived from Stan Modeling Language
+   * User's Guide and Reference Manual.
+   *
+   * @param y         Observed values
+   * @return Occasion of last capture
+   */
     for (k_rev in 0:(size(y_i) - 1)) {
-      int k;
-      k <- size(y_i) - k_rev;
+      // Compound declaration was enabled in Stan 2.13
+      int k = size(y_i) - k_rev;
+      //      int k;
```

Comment:
You don't need the commented out code any more or the comment about compound declare/define.